bluemira/display/plotter.py

```diff
@@ -717,7 +717,7 @@ def _create_plotters(self, comp: Component) -> Iterator[BasePlotter]:
                 if self.options._external_options:
                     options = comp.plot_options
                 else:
-                    options = self.options
+                    options = comp.plot_options
             else:
                 options = comp.plot_options
```

Comment:
@je-cook I did this because whatever I wanted to work wasn't. But now this logic is completely odd-looking :/